### DISCUSSION TITLE
How to delete clusters from definition library

### DISCUSSION THREAD

**Discussion by mariomodest** [nicolaasburgers/definition-library-releases#40] [Q&A]
I'm trying to delete clusters that were published through the definition library so that they don't show up in the library anymore. I tried deleting the file in github but the cluster still exists in the library and I can even still import the cluster. Does anyone have experience with that?

**Comment by nicolaasburgers** [comment]
Funny you should say that - the next version (currently in final testing stage) has that very feature!
You'll see a new "remove definition" button next to the "Change Maintainer" button.
Stay tuned for the new version and I'll update this thread when it's out.

**Reply by mariomodest** [reply]
Perfect thanks!

**Reply by nicolaasburgers** [reply]
Sorry for the delay in getting the new version out - it's still on its way, I got so excited about a web viewer for DefinitionLibrary that I took a detour to create a prototype and video for that (which I'll add a discussion item for).
